packages/teleterm/src/ui/services/workspacesService/workspacesService.ts

```diff
@@ -1,13 +1,25 @@
 import { useStore } from 'shared/libs/stores';
 
 import { isEqual } from 'lodash';
+import isPast from 'date-fns/isPast';
+
+// adding requests to assumed adds already 'made' requests that fit our tables
+import { AccessRequest as UiAccessRequest } from 'e-teleport/services/workflow';
```

Comment:
Is this rename to `UiAccessRequest` needed?

Reply:
it was... until a refactor. Thank you for catching